lizard_auth_client/migrations/0005_auto_20181115_1249.py

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+
+def delete_duplicate_uors(apps, schema_editor):
+    """Delete duplicate UserOrganisationRoles."""
+    UOR = apps.get_model('lizard_auth_client', 'UserOrganisationRole')
+    for row in UOR.objects.all():
```

Comment:
Je loopt rijen te deleten nadat deze "hoofdquery" ze al had opgehaald. Ik vroeg me af of dat goed gaat.
Na wat nadenken denk ik "ja, volgens mij gaat het wel goed".
Jij vindt het kennelijk ook, dus: ok :-)